frontend/app/components/Home.tsx

```diff
@@ -105,7 +105,7 @@ function Home (props: HomeProps) {
         </div>
       </MainContainer>
       <Typography component='footer' sx={{ textAlign: 'center' }}>
-        Copyright © 2022 The Regents of the University of Michigan
+        Copyright © {new Date().getFullYear()} The Regents of the University of Michigan
```

Comment:
I think it's good to update this but I feel like it should remain a static number. I've read debate out there both ways, but this way would pull from the client date which can be changed. I don't know if that matters. If we wanted to put this in backend home.html template that would be different. Do we have any existing apps with dynamic updates?

I'd probably just update it to 2024 for now and create a new issue to investigate this for best practice.

Reply:
OK, I will make it static for now.

Reply:
Created an issue https://github.com/tl-its-umich-edu/instructor-productivity-tools/issues/360